src/app/interceptors/http-params-interceptor.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import {
+  HttpEvent, HttpInterceptor, HttpHandler, HttpRequest
+} from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class HttpParamInterceptor implements HttpInterceptor {
+  intercept(req: HttpRequest<any>, next: HttpHandler):
+    Observable<HttpEvent<any>> {
+    if (!req.headers.has('Content-Type')) {
```

Comment:
ten `if` wg mnie jest niepotrzebny. Znalazłeś jakiś przypadek, w którym nasza apka ustawia wcześniej content-type ?

Reply:
aaaa jak testowalem to dawalem ifa  zeby ozbaczyc czy faktycznie  dodaje już zmieniam :D